Dockerfile

```diff
@@ -46,7 +57,7 @@ COPY --chown=jitsi:jitsi docker/run-skynet.sh /opt/
 RUN \
     apt-dpkg-wrap apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F23C5A6CF475977595C89F51BA6932366A755776 && \
     apt-dpkg-wrap apt-get update && \
-    apt-dpkg-wrap apt-get install -y python3.11 python3.11-venv tini libgomp1 && \
+    apt-dpkg-wrap apt-get install -y python3.11 python3.11-venv tini libgomp1 libcurl4-openssl-dev && \
```

Comment:
For prod you don't want the `-dev` library, just the runtime.

Reply:
that's a relic from some different attempt to make the server work, will remove